Userland/Libraries/LibC/sys/arch/regs.h

```diff
@@ -8,8 +8,6 @@
 
 #include <AK/Platform.h>
 
-#if ARCH(I386)
-#    include "i386/regs.h"
```

Comment:
wait wat. arch/i386/regs.h had #if ARCH(I386)  checks? since x86_64/regs.h exists, the i386 file should be removed.

Reply:
it was easy to mess around with that—while not having enough mana myself (sorcerersʼ apprentice!) iʼm glad the heavy-lifting is done along!

thanks mr. supercomputer7! 🙇🏻‍♂️